src/test/resources/journeys.conf

```diff
@@ -140,6 +140,15 @@ journeys {
       run-if = ["journeyFlows"]
     }
 
+    income-tax-tailor-return-journeys = {
+          description = "Income Tax Tailor Return - Agents/Individuals "
+          load = 0.2
+          parts = [
+            individual-tailor-return,
+            agent-tailor-return
+          ]
+          run-if = ["journeyFlows/"]
```

Comment:
@SamuelTwumBarima I think you need to remove the `/` at the end of the `run-if`. Also can you format the indentation on this format.

Reply:
ahh good spot, I will make a change now, thanks